app/views/layouts/application.html.erb

```diff
@@ -11,6 +21,7 @@
     <%= tag :link, rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' %>
     <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload', media: 'all' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
+    <%= javascript_pack_tag 'google_analytics' %>
```

Comment:
I don't think this needs to be off in its own pack file? It will add another roundtrip to the server with associate latency for 7 lines of JS? Could it go in the application pack file instead?

Reply:
Sounds good